components/automate-gateway/api/api_test.go

```diff
@@ -12,12 +12,7 @@ import (
 	_ "github.com/chef/automate/api/external/nodes"
 	_ "github.com/chef/automate/api/external/nodes/manager"
 	_ "github.com/chef/automate/api/external/secrets"
-	_ "github.com/chef/automate/components/automate-gateway/api/auth/teams"
 	_ "github.com/chef/automate/components/automate-gateway/api/authz"
-	_ "github.com/chef/automate/api/external/compliance/profiles"
-	_ "github.com/chef/automate/api/external/compliance/reporting"
-	_ "github.com/chef/automate/api/external/compliance/reporting/stats"
-	_ "github.com/chef/automate/api/external/compliance/scanner/jobs"
```

Comment:
Interesting diff. Were none of these being used?

Reply:
ooo good catch i think that was some overzealous rebase deletion